Applications/WebPortal/WebApp/library/source/components/text_field.tsx

```diff
@@ -68,6 +74,11 @@ export class TextField extends React.Component<Properties> {
           this.props.className}/>);
   }
 
+  /** Gives focus to the component. */
```

Comment:
My preference would be to add an optional ```boolean``` prop called ```autofocus``` to ```TextField``` and then use that from with the group modal.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefautofocus

Reply:
I don't think I can get it to work that way.